packages/lab/stories/input-next/input-next.doc.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { InputNext } from "@salt-ds/lab";
+import { Meta, Canvas, ArgsTable } from "@storybook/addon-docs";
+import {
+  CSSClassTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+import { Link } from "@salt-ds/core";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Input Next"
+  component={InputNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Input Next
+
+Input provides an editable field in which users can enter text and numeric values.
+
+# Examples
+
+## Default
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line.
+
+<Canvas>
+  <Story id="lab-input-next--default" />
+</Canvas>
+
+## Controlled
+
+You can control Input with the `value` and `onChange` props. Learn more about <Link href="https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components">controlled vs. uncontrolled components in the official React docs</Link>.
+
+<Canvas>
+  <Story id="lab-input-next--controlled" />
+</Canvas>
+
+## Variants
+
+Input has two variants, primary (default) and secondary.
+
+<Canvas>
+  <Story id="lab-input-next--variants" />
+</Canvas>
+
+## Disabled
+
+You can set Input to a disabled state, with no resultant action when the user interacts with it.
+
+<Canvas>
+  <Story id="lab-input-next--disabled" />
+</Canvas>
+
+## Read-only
+
+You can set Input to a read-only state. The value text can’t be edited in this state, but it can be highlighted and copied.
+
+<Canvas>
+  <Story id="lab-input-next--readonly" />
+</Canvas>
+
+For accessibility purposes, we recommend that you add a marker for empty read-only fields via the `emptyReadOnlyMarker` prop.
+
+<Canvas>
+  <Story id="lab-input—next--empty-readonly-marker" />
+</Canvas>
+
+## Text Alignment
+
+Text alignment is left by default. This can be changed via the `textAlign` prop to “center” or “right”.
```

Comment:
This is a minor thing, but are we using ” and ’ everywhere now?

Reply:
That is the current preferred style.